frontend/src/components/OtherImplementations.tsx

```diff
@@ -38,22 +41,44 @@ export const OtherImplementations = ({ otherImplementationsData }: Props) => {
           {(props) => (
             <Popover id="popover-basic" {...props}>
               <Popover.Body>
-                {Object.entries(otherImplementationsData).map(([id, impl]) => {
-                  const implementationPath = getImplementationPath(id);
-                  return (
-                    <p key={id}>
-                      <NavLink
-                        style={{ fontSize: "1rem", fontWeight: "bold" }}
-                        to={`/implementations/${implementationPath}`}
-                      >
-                        {impl.name}
-                      </NavLink>
-                      <span className="ps-1 text-body-secondary fw-bold">
-                        {mapLanguage(impl.language)}
-                      </span>
-                    </p>
-                  );
-                })}
+                <Container className="p-0">
+                  <Row className="flex-column">
+                    {otherImplementationsDataArray.map(([id, impl], index) => {
+                      const implementationPath = getImplementationPath(id);
+                      const latestSupportedDialect =
+                        getLatestSupportedDialect(impl);
+                      return (
+                        <Col
+                          key={id}
+                          className={
+                            index === otherImplementationsDataArray.length - 1
```

Comment:
I kind of understand what are you doing here, but for such cases flexbox comes in handy. I believe `<Col>` component already has `display: flex` so you only need to set gap - always prefer spacing elements in a list with a `gap` instead of `margin`

Reply:
What is the exact keyword for `gap` here ? I tried doing `gap-2` and also `flex-gap-2` both didn't work

Reply:
Ohh gap works on the grid ig then. Will do the changes 👍